docs/docs/getting-started/first-steps.md

```diff
@@ -40,181 +40,51 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 2. Fill in your cloud provider specific information.
 
     <tabs groupId="csp">
-    <tabItem value="azure" label="Azure (CLI)">
 
-    You need several resources for the cluster. You can use the following `az` script to create them:
+    <tabItem value="azure" label="Azure">
 
     ```bash
-    RESOURCE_GROUP=constellation # enter name of new resource group for your cluster here
-    LOCATION=westus # enter location of resources here
-    SUBSCRIPTION_ID=$(az account show --query id --out tsv)
-    SERVICE_PRINCIPAL_NAME=constell
-    az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
-    az group create --name "${RESOURCE_GROUP}-identity" --location "${LOCATION}"
-    az ad sp create-for-rbac -n "${SERVICE_PRINCIPAL_NAME}" --role Owner --scopes "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}" | tee azureServiceAccountKey.json
-    az identity create -g "${RESOURCE_GROUP}-identity" -n "${SERVICE_PRINCIPAL_NAME}"
-    identityID=$(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query principalId --out tsv)
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Virtual Machine Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Application Insights Component Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    echo "subscription: ${SUBSCRIPTION_ID}
-    tenant: $(az account show --query tenantId -o tsv)
-    location: ${LOCATION}
-    resourceGroup: ${RESOURCE_GROUP}
-    userAssignedIdentity: $(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query id --out tsv)
-    appClientID: $(jq -r '.appId' azureServiceAccountKey.json)
-    clientSecretValue: $(jq -r '.password' azureServiceAccountKey.json)"
+    constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
     ```
 
-    Fill the values produced by the script into your configuration file.
-
-    :::tip
-
-    Alternatively, you can leave `clientSecretValue` empty and provide the secret via the `CONSTELL_AZURE_CLIENT_SECRET_VALUE` environment variable.
-
+    :::info
+    This command creates IAM configuration on the Azure region `westus` using a new resource group `constellTest` and a new service principal `spTest`.
```

Comment:
using would imply to me that I am responsible for creating the SP so that `iam create` can use it.
```suggestion
    This command creates IAM configuration on the Azure region `westus` creating a new resource group `constellTest` and a new service principal `spTest`.
```